src/GraphQL.Harness.Tests/SuccessResultAssertion.cs

```diff
@@ -1,40 +1,46 @@
+using System.Linq;
 using Alba;
-using GraphQL.NewtonsoftJson;
+using GraphQL.SystemTextJson;
```

Comment:
```suggestion
```

Reply:
This is needed for the `actualResultJson.ToDictionary();` as that is an extension method. It's not a great name for that method though, but I think that was legacy and I'd keep it as is. Certainly a bit confusing.